libs/counterfactuals/src/lib/CounterfactualList.tsx

```diff
@@ -173,9 +177,9 @@ export class CounterfactualList extends React.Component<
         const optionIndex = dropdownOption?.data.categoricalOptions.findIndex(
           (feature: IComboBoxOption) => feature.key === data[k]
         );
-        this.props.setCustomRowProperty(`Data${keyIndex}`, true, optionIndex);
+        this.props.setCustomRowProperty(columnKey, true, optionIndex);
```

Comment:
Does this mean we're no longer adding "Data" as a prefix? Just wondering if that affects backward compatibility in some way.